cv_next/server/api/cvs.ts

```diff
@@ -1,277 +1,165 @@
 import "server-only"
 
-import Helper from "@/server/base/helper"
 import MyLogger from "@/server/base/logger"
 import Categories from "@/types/models/categories"
-import CvModel from "@/types/models/cv"
-import {
-  DocumentData,
-  Query,
-  QueryDocumentSnapshot,
-  QueryFieldFilterConstraint,
-  collection,
-  limit,
-  orderBy,
-  query,
-  startAfter,
-  where,
-  documentId,
-  CollectionReference,
-} from "firebase/firestore"
-import Definitions from "../base/definitions"
-import FirebaseHelper from "./firebaseHelper"
-import { DbOperationResult, ErrorReasons, RateLimitError } from "./utils"
+import Definitions from "../../lib/definitions"
+import SupabaseHelper from "./supabaseHelper"
+import { PostgrestError } from "@supabase/supabase-js"
 
 const queryLimit: number = Definitions.CVS_PER_PAGE
```

Comment:
I don't think this should (stay?) be defined globally in this file; just use the Definitions constant in the used functions. Perhaps add a query limit parameter to optionally bypass the default